src/engine/pieces/pawn.ts

```diff
@@ -1,13 +1,28 @@
 import Piece from './piece';
 import Player from '../player';
 import Board from '../board';
+import Square from '../square';
 
 export default class Pawn extends Piece {
     public constructor(player: Player) {
         super(player);
     }
 
     public getAvailableMoves(board: Board) {
-        return new Array(0);
+
+        let moveArr: Square[] = []; // Array of squares to move to
+        let currSquare: Square = board.findPiece(this);
```

Comment:
We'll come to some deeper code style discussions tomorrow, but abbreviations like `currSquare` are often unnecessary - we should be bold enough to use the extra characters and write `currentSquare` or even better `squareOfCurrentPiece`!